src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -101,20 +101,6 @@ case class LocalBackend(actorSystem: ActorSystem) extends Backend with SharedFil
     }
   } map CompletedExecutionHandle
 
-  def useCachedCall(cachedBackendCall: BackendCall, backendCall: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] = Future {
```

Comment:
Just a heads up there are going to be some conflicts with my branch on these changes